components/BottomSheet/src/MDCBottomSheetController.h

```diff
@@ -45,6 +45,11 @@
  */
 @property(nonatomic, weak, nullable) UIScrollView *trackingScrollView;
 
+/**
+ When set to true, the bottom sheet controller can't be dismissed by tapping outside of sheet area.
```

Comment:
This comment is documenting the opposite of what I think the behavior is.

Reply:
Looks like the boolean's implementation is the inverse of its name - either change the implementation or the name.